spacialgaze-plugins/profile.js

```diff
@@ -321,10 +309,10 @@ exports.commands = {
 				let profile = '';
 				profile += showBadges(toId(username));
 				profile += '<img src="' + avatar + '" height="80" width="80" align="left">';
-				if (!getFlag(toId(username))) {
+				if (!SG.getFlag(toId(username))) {
```

Comment:
This is looking for a possible `return false` if the new function can't return false (and if its not needed **test this please**) then this if statement is unneeded.